distributor-api-v1/deprecations/payment-card-input.md

```diff
@@ -0,0 +1,154 @@
+                },
+                "Pricing": "Gross",
+                "PrivacyPolicyUrl": {
+                    "en-US": "https://localhost/en"
+                },
+                "Products": [
+                    {
+                        "Id": "1627aea5-8e0a-4371-9022-9b504344e724",
+                        "Amounts": {
+                            "EUR": {
+                                "GrossValue": 5.00,
+                                "NetValue": 4.5,
+                                "TaxValues": [
+                                    {
+                                        "TaxRateCode": "DE-R",
+                                        "Value": 0.5
+                                    }
+                                ]
+                            },
+                            "CZK": {
+                                "GrossValue": 150.00,
+                                "NetValue": 140.00,
+                                "TaxValues": [
+                                    {
+                                        "TaxRateCode": "DE-R",
+                                        "Value": 10.00
+                                    }
+                                ]
+                            }
+                        },
+                        "CategoryId": null,
+                        "Charging": "Once",
+                        "Description": {
+                            "en-US": "Continental breakfast served in the morning."
+                        },
+                        "ImageId": "1627aea5-8e0a-4371-9022-9b504344e724",
+                        "IncludedByDefault": false,
+                        "Name": {
+                            "en-US": "Breakfast"
+                        },
+                        "Ordering": 0,
+                        "Posting": "Once",
+                        "RelativePrice": null
+                    }
+                ],
+                "TaxEnvironmentCode": "NL",
+                "Telephone": "030 6926666",
+                "TermsAndConditionsUrl": {
+                    "en-US": "https://website.com/terms-and-conditions.html"
+                }
+            },
+            "OnlineTravelAgencies": [],
+-           "PaymentCardInput": "NotRequested",
++           "PaymentCardRequirement": "NotRequired",
+            "RequiredFields": [],
+            "ServiceId": "c1eec12a-1101-4bg6-ad24-e48f8dlpb9ee"
+        }
+    ],
+    "CurrencyCode": null,
+    "CurrencyCodes": [],
+    "DisplayVoucherCode": false,
+    "EndDateOffset": null,
+    "GtmContainerId": "",
+    "IntroVideoUrl": "",
+    "LanguageCode": null,
+    "NowUtc": "2020-04-09T07:18:48Z",
+    "PrimaryColor": "",
+    "StartDateOffset": null,
+    "Theme": null,
+    "VoucherCode": ""
+}
+```
+
+### PaymentCardInput
+* `NotRequested` - Payment card info is not requested.
+* `Requested` - Payment card info is requested, but not validated.
+* `Required` - Payment card info is requested and validated.
```

Comment:
Not sure whether to mention the old version here. It is valuable to see what the options was, but I would somehow mark it as "invalid" maybe a ~~stiketrough~~ might do the job?

@vitasamek what do you think?